packages/core/src/Stepper.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { themeGet } from 'styled-system'
+import Flex from './Flex'
+import Step from './Step'
+
+const StyledFlex = styled(Flex)`
```

Comment:
I think the name could be `StepperFlex`, since we know the `styled` API is implied and the term "StepperFlex" can relate the building block `Flex` with the purpose `Stepper` together.